src/content/documentation/develop/developing-extensions-for-firefox-for-android.md

```diff
@@ -39,8 +41,43 @@ However, instructions are provided for the steps you need to take should you cho
 
 {% capture content_with_toc %}
 
+## Testing instructions for Firefox for Android 79+ (Fenix)
+:::note alert
+Persistent loading is currently not supported. These instructions are for temporary loading.
+:::
+
+- Install or update [web-ext](https://github.com/mozilla/web-ext) to version 4.1.0 or later.
+
+- Install or update to the latest version of [Firefox Nightly for Android](https://play.google.com/store/apps/details?id=org.mozilla.fenix).
+
+- Make sure you have adb installed ([Linux](https://dl.google.com/android/repository/platform-tools-latest-linux.zip), [Mac](https://dl.google.com/android/repository/platform-tools-latest-darwin.zip), [Windows](https://dl.google.com/android/repository/platform-tools-latest-windows.zip)) and in your PATH.
+
+- Connect your Android device via USB.
+
+- Make sure you have [Android's USB debugging](https://developer.android.com/studio/debug/dev-options) enabled. You can run `adb devices` to verify.
+
+- In the settings for Firefox Nightly, enable "Remote debugging via USB."
+
+- In the unzipped directory of your extension, run `web-ext run -t firefox-android` and follow the instructions on screen to make sure you select the right device. Select `org.mozilla.fenix` as a target.
```

Comment:
in the `web-ext run --help` output we call `target` the `-t firefox-android` option, and so it may be better to call this "apk" or "Firefox apk" (which will better match the name of the related cli option, `--firefox-apk`).

Reply:
Sorry, I don't understand what to change. :(